core/app/[locale]/(default)/login/register-customer/page.tsx

```diff
@@ -63,9 +64,11 @@ export default async function RegisterCustomer({ params: { locale } }: Props) {
           countries={countries}
           customerFields={customerFields}
           defaultCountry={{ entityId, code, states: statesOrProvinces ?? [] }}
-          reCaptchaSettings={reCaptchaSettings}
+          reCaptchaSettings={bypassReCaptcha(reCaptchaSettings)}
         />
       </NextIntlClientProvider>
     </div>
   );
 }
+
+export const runtime = 'edge';
```

Comment:
Can you please confirm if this is part of this PR and how is it being used?

Reply:
@avattipalli yes, this is part of this PR - it is needed to bypass recaptcha in automated tests. We cannot disable it entirely since our Demo store is linked to that CP. This part of code was initially added by Chance to my previous PR [#1015](https://github.com/bigcommerce/catalyst/pull/1015/files#diff-1cd466ab08fc77f380166434d307c7274001f261e772d40caca89fdff2532612) as a result of our Slack discussion on ways to bypass recaptcha.